requests-for-comments/2023-10-27-base3000-launch.md

```diff
@@ -0,0 +1,34 @@
+# Base 3000 & Notebooks launch
+
+We want to launch the ‘first version’ of PostHog 3000 after the next sprint, which means the week starting November 13th (+ 1 week if needed). We have decided to **include Notebooks** in this launch, so that we can have a big launch that combines functionality and design changes.
+
+## Scope for launch
+
+- Dark Mode
+- Base 3000 [Check this mega issue for details](https://github.com/PostHog/posthog/issues/18089)
+    
+<img width="674" alt="Screenshot 2023-10-27 at 10 38 50" src="https://github.com/PostHog/meta/assets/14750837/abb9e559-91ea-408b-a8fd-5105f3c4c626">
+
+## Out of scope for launch (amongst other things)
+
+- Sidebar (Index view)
+- Canvas-powered dashboards
+- Query builder
+- [Check this mega issue for details + the full list](https://github.com/PostHog/posthog/issues/18089)
+
+## Rollout plan
+
+- [x]  Now: Roll out [Notebooks](https://app.posthog.com/feature_flags/8734) to PostHog Team
+- [ ]  Now: Roll out [PostHog 3000 (Base 3000 scope)](https://app.posthog.com/feature_flags/8472) to PostHog Team
+- [ ]  October 30th: Roll out Notebooks Beta to 100% of users, to get some early feedback
```

Comment:
Is the 100% here a typo? Since we have roll out notebooks to 100% of users further down again. Otherwise I might not understand how notebooks beta and notebooks differ from each other.

Reply:
I too assume this is a typo. If it's less than 100%, I'd like to know how we're targeting these users.

Reply:
Ah good spot. I first had listed roll out to 100% for the 'big' launch, and then I read @joethreepwood's Notebooks launch RFC where he proposed a beta launch of notebooks first, so I incorporated this, because I think it's a good idea. 
Notebooks is (almost) ready and we can expose it in the UI (with a beta tag), getting some people to use it, and then announce it a couple of weeks later, I don't think there is a conflict here.

Reply:
Note: I don't think we are beta-ready by Monday, more likely middle of next week